tests/foreman/cli/test_contentview.py

```diff
@@ -5136,7 +5139,49 @@ class ContentViewFileRepoTestCase(CLITestCase):
     """Specific tests for Content Views with File Repositories containing
     arbitrary files
     """
-    @stubbed()
+    @classmethod
+    def setUpClass(cls):
+        """Create a product and an org which can be re-used in tests."""
+        super(ContentViewFileRepoTestCase, cls).setUpClass()
+        cls.org = make_org()
+        cls.product = make_product({'organization-id': cls.org['id']})
+
+    def _make_file_repository_upload_contents(self, options=None):
+        """Makes a new File repository, Upload File/Multiple Files
+        and asserts its success """
+        if options is None:
+            options[u'product-id'] = self.product['id']
+
+        options_list = (u'product-id', u'content-type', u'url', u'name')
+        if None in [options.get(keys) for keys in options_list]:
```

Comment:
I think this was mentioned before but is it intentional that if any of these options is not given then it uses all defaults?